.github/workflows/release-final.yml

```diff
@@ -1,6 +1,16 @@
 name: "[Release] Publish packages and apps"
 on:
   workflow_dispatch:
+    inputs:
+      app:
+        description: "App to release"
+        required: true
```

Comment:
What if we just wanna release libs and no apps? Maybe it would be best to make it optional.